### ISSUE TITLE
Fix install requirements

### ISSUE THREAD

**Issue by YannickDieter** [SiLab-Bonn/beam_telescope_analysis#20]
Install requirements for Numba and PyLandau cannot be fulfilled simultaneously at the moment. Numba requires numpy<1.22,>=1.18, but PyLandau requires numpy>=1.22.

``` console
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
Successfully installed numpy-1.22.3 pylandau-2.2.0 scipy-1.8.0
numba 0.55.1 requires numpy<1.22,>=1.18, but you have numpy 1.22.3 which is incompatible
```

**Comment by cbespin** [comment]
Yes, this is known from pylandau. The underlying issue is the numpy support in numba. Since numpy changed its C headers from 1.21 to 1.22, it is not straightforward to compile pylandau with numpy 1.21.

Solution (for now at least) could be to use numba for pylandau instead of cython. I will see what I can do about that

**Comment by cbespin** [comment]
This should be solved for now with [pylandau 2.2.1](https://github.com/SiLab-Bonn/pylandau/releases/tag/v2.2.1). Feel free to reopen if this does not work for you.

**Comment by YannickDieter** [comment]
It is working. Thanks for fixing it so fast!
